routers/private/serv.go

```diff
@@ -317,22 +329,11 @@ func ServCommand(ctx *context.PrivateContext) {
 		} else {
 			// Because of the special ref "refs/for" we will need to delay write permission check
 			if git.SupportProcReceive && unitType == unit.TypeCode {
+				// XXX this makes no sense? https://github.com/go-gitea/gitea/issues/18515
```

Comment:
Yeah totally!!

Since they closed it should I just drop this comment since it doesn't go anywhere useful? Or is the attempt useful context for us for the future do you think?